ReactiveCocoaTests/UIKit/UIKeyboardSpec.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+import ReactiveSwift
+import ReactiveCocoa
+import Quick
+import Nimble
+
+class UIKeyboardSpec: QuickSpec {
+	override func spec() {
+		describe("UIView.Type.reactive.keyboardChange") {
+			it("should emit a `value` event when the notification is posted") {
+				var context: KeyboardChangeContext?
+
+				UIView.reactive.keyboardChange.observeValues { c in
+					context = c
+				}
+
+				let dummyInfo: [AnyHashable: Any] = [
+					UIKeyboardFrameBeginUserInfoKey: CGRect(x: 10, y: 10, width: 10, height: 10),
+					UIKeyboardFrameEndUserInfoKey: CGRect(x: 20, y: 20, width: 20, height: 20),
+					UIKeyboardAnimationDurationUserInfoKey: 1.0,
+					UIKeyboardAnimationCurveUserInfoKey: UIViewAnimationCurve.easeInOut
+				]
+
+				NotificationCenter.default.post(name: .UIKeyboardWillChangeFrame,
```

Comment:
It would then have to be exposed as an extension to `NotificationCenter` instances, I'd think.